src/SFA.DAS.EmployerAccounts/Interfaces/IClientContentApiClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerAccounts.Queries.GetClientContent;
+
+namespace SFA.DAS.EmployerAccounts.Interfaces
+{
+    public interface IClientContentApiClient
+    {
+        Task<string> GetContentByClientId(ContentType type, string clientId);
```

Comment:
Could the method just be Get ?  since that is what it is doing.  It's explicit that it's by ClientId since that is a required parameter.